neutone_sdk/utils.py

```diff
@@ -165,14 +177,17 @@ def save_neutone_model(
         loaded_model.reset()
         loaded_model.is_resampling()
 
-        log.info("Testing offline mode...")
-        for input_sample in get_default_audio_samples():
-            offline_sr = input_sample.sr
-            offline_bs = 4096
-            loaded_model.set_daw_sample_rate_and_buffer_size(offline_sr, offline_bs)
-            offline_params = tr.rand((MAX_N_PARAMS, input_sample.audio.size(1)))
-            offline_rendered_sample = loaded_model.forward_offline(input_sample.audio, offline_params)
-            break  # Only test one sample to reduce export time
+        if test_offline_mode:
+            log.info("Testing offline mode...")
+            for input_sample in get_default_audio_samples():
+                offline_sr = input_sample.sr
+                offline_bs = 4096
+                loaded_model.set_daw_sample_rate_and_buffer_size(offline_sr, offline_bs)
+                offline_params = tr.rand((MAX_N_PARAMS, input_sample.audio.size(1)))
+                offline_rendered_sample = loaded_model.forward_offline(
+                    input_sample.audio, offline_params
+                )
+                break  # Only test one sample to reduce export time
```

Comment:
This actually broke the export for me in the previous version but I didn't get around to pushing the changes. Could you add
```python
log.info("Reloading model...")
loaded_model, loaded_metadata = load_neutone_model(root_dir / "model.nm")
```
to the submission block below? Line 198 or 195 or so.
Models with state that don't implement reset properly cannot pass the checks below. Ideally we'd enforce it but I think it's hard enough to build a model as is and the reset doesn't seem to be that important on the plugin side.